src/test/java/org/codeforamerica/shiba/AbstractBasePageTest.java

```diff
@@ -140,4 +154,277 @@ protected void fillOutContactInfo() {
         testPage.enter("phoneNumber", "7234567890");
         testPage.enter("phoneOrEmail", "Text me");
     }
+
+    protected Boolean allPdfsHaveBeenDownloaded() {
+        File[] listFiles = path.toFile().listFiles();
+        List<String> documentNames = Arrays.stream(listFiles).map(File::getName).collect(Collectors.toList());
+
+        Function<Document, Boolean> expectedPdfExists = expectedPdfName -> documentNames.stream().anyMatch(documentName ->
+                documentName.contains("_MNB_") && documentName.endsWith(".pdf") &&
+                        documentName.contains(expectedPdfName.toString())
+        );
+        return List.of(CAF, CCAP).stream().allMatch(expectedPdfExists::apply);
+    }
```

Comment:
This was an actual refactor. I extracted this function from `userCanCompleteTheNonExpeditedFlowAndCanDownloadPdfsAndShibaShouldCaptureMetricsAfterApplicationIsCompleted` and simplified some of the stream method chains. I also extracted the `expectedPdfExists` function just for readability.

Reply:
(No behavior change)